README.md

```diff
@@ -763,6 +764,44 @@ Example output:
 }
 ```
 
+### Uploading Manifests
+
+ArDrive supports the creation of Arweave manifests using any of your PUBLIC folders or drives. This manifest will be created at the root of specified folder or drive just as any other ArFS file entity:
```

Comment:
> just as any other ArFS file entity:
That's a little confusing; you can put ordinary files wherever you want in a drive. Perhaps rephrase as "will reside in the root folder of the drive they describe." Which actually brings up an interesting point - what happens if you move  a manifest file? Does it have any negative impacts on its functionality?

Reply:
I don't foresee any negative impacts with moving the manifest file. The dataTxId is the main pointer of the manifest and that wouldn't be effected